src/dotnet.json

```diff
@@ -34,5 +34,6 @@
     "minimum": "1.0.6",
     "maximum": ""
   },
-  "lspLangId": "csharp"
-}
+  "lspLangId": "csharp",
+  "gRPCSupport": true
```

Comment:
can we use `"capabilities" : []` format instead of top level toggle? we do this for html-report.

Reply:
If we change it, then we have to make changes in gauge (it's already implemented and expects the toggle to be at the top level). We also have to make changes in runner which has already implemented this toggle.

I think we can handle this is a separate issue. That way we can move existing toggle/config (multithreaded, lspLangID) to capabilities/properties as well.

Thoughts?

Reply:
let's leave it as it is for now.